### PR TITLE
Add failing test fixture for PrivatizeFinalClassPropertyRector

### PR DESCRIPTION
**Pull request by gisostallenberg**
# Failing Test for PrivatizeFinalClassPropertyRector

Based on https://getrector.org/demo/1ec5cc84-5490-64e6-8e1a-9f08fa5679ef

### CODE REVIEW

File: rules-tests/Privatization/Rector/Property/PrivatizeFinalClassPropertyRector/Fixture/keep_used_protected.php.inc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    public function __construct()
+    {
+        if (isset($this->options)) {
+            $this->abstractOptions['options'] = $this->options;
+        }
+    }
+}
+
+namespace Rector\Tests\Privatization\Rector\Property\PrivatizeFinalClassPropertyRector\Fixture;
```
Comment (TomasVotruba) [inline rules-tests/Privatization/Rector/Property/PrivatizeFinalClassPropertyRector/Fixture/keep_used_protected.php.inc]: There should be only one namespace in the file.

Comment (gisostallenberg) [reply rules-tests/Privatization/Rector/Property/PrivatizeFinalClassPropertyRector/Fixture/keep_used_protected.php.inc]: I missed that, will fix. This is produced by the button Create a Test on https://getrector.org/demo.

**Comment by TomasVotruba** [comment]
Looks good, thank you :+1:

**Comment by samsonasik** [comment]
@gisostallenberg I cherry-picked your commit at PR https://github.com/rectorphp/rector-src/pull/1501

**Comment by gisostallenberg** [comment]
> @gisostallenberg I cherry-picked your commit at PR #1501

Nice!

**Pull request closed without merging**